packages/bpk-component-radio/src/BpkRadio.js

```diff
@@ -41,18 +40,15 @@ const BpkRadio = props => {
   // treated as neither valid nor invalid
   const isInvalid = valid === false;
 
-  if (white) {
-    classNames.push(getClassName('bpk-radio--white'));
-  }
-  if (disabled) {
-    classNames.push(getClassName('bpk-radio--disabled'));
-  }
-  if (isInvalid) {
-    classNames.push(getClassName('bpk-radio--invalid'));
-  }
-  if (className) {
-    classNames.push(className);
-  }
+  const classNames = [
+    getClassName(
+      'bpk-radio',
+      white && 'bpk-radio--white',
```

Comment:
You could also simplify this a bit with `bpk-radio--${style}`. Probably unnecessary though 🤷